big_tests/tests/graphql_sse_SUITE.erl

```diff
@@ -145,6 +145,10 @@ sse_should_not_get_timeout(Config) ->
         timer:sleep(2000),
         escalus:send(Bob, escalus_stanza:chat(From, To, <<"Hello again!">>)),
         sse_helper:wait_for_event(Stream),
+        timer:sleep(2000),
```

Comment:
Do we need to have these sleeps?
We could also made a separate testcase like `sse_works_with_long_messages`.

And run sse_should_not_get_timeout and sse_works_with_long_messages in parallel.